airflow/www/views.py

```diff
@@ -543,7 +544,8 @@ class AirflowBaseView(BaseView):
     }
 
     if not conf.getboolean('core', 'unit_test_mode'):
-        extra_args['sqlite_warning'] = settings.Session.bind.dialect.name == 'sqlite'
+        session: sqlalchemy.orm.session.Session = settings.Session
+        extra_args['sqlite_warning'] = session.bind.dialect.name == 'sqlite'
```

Comment:
Although it is likely better to fix this in `settings` instead since I’d expect this very issue to pop up for other modules.

I think changing `Session: Optional[SASession] = None` there to just `Session: SASession` would fix the issue.

Reply:
Ok. I will try it